crates/apollo-parser/src/parser/grammar/enum_.rs

```diff
@@ -26,7 +25,7 @@ pub(crate) fn enum_type_definition(p: &mut Parser) {
     }
 
     if let Some(T![@]) = p.peek() {
-        directive::directives(p);
+        directive::directives(p, Constness::NotConst);
```

Comment:
should be const on enum type definition
```suggestion
        directive::directives(p, Constness::Const);
```

Reply:
Good catch!